Makefile

```diff
@@ -8,7 +8,7 @@ zip: deps build
 
 new:
 	mkdir $(adapter)
-	cp -r example/ $(adapter)
+	cp -rT example/ $(adapter)
```

Comment:
This is what my macOS `cp` implementation states for `-r`: 

> Historic versions of the cp utility had a -r option.  This implementation supports that option; however, its use is strongly discouraged, as it does not correctly copy special files, symbolic links, or fifo's.

And looks like I do not have a `-T` option.

Maybe just `-R` would work?

Reply:
If you use `cp -R example/* $(adapter)` does that bring the `.eslintrc.js` with it? Looking at the macOS man page for `cp` I would guess it won't, but worth a try.

We could also add a line to just manually copy the `.eslintrc.js`

Reply:
- `cp -R example/* $(adapter)` does NOT copy the `.eslintrc.js`
- `cp -R example/ $(adapter)` does copy the `.eslintrc.js`